pages/shopping/index.js

```diff
@@ -1,16 +1,18 @@
-import StyledList from "@/components/Styled/StyledList";
-import AddButton from "@/components/Styled/AddButton";
+import {
+  Select,
+  H2,
+  Button,
+  List,
+  ListItem,
+  Spacer,
+} from "@/components//Styled/Styled";
```

Comment:
```suggestion
} from "@/components/Styled/Styled";
```
Fix the import here :)